api/types/networking.go

```diff
@@ -56,6 +56,15 @@ type ClusterNetworkingConfig interface {
 
 	// SetSessionControlTimeout sets the session control timeout.
 	SetSessionControlTimeout(t time.Duration)
+
+	// GetIdleTimeoutMessage fetches the message to be sent to the client in
```

Comment:
I think this is fine, but let's talk about this today in the context of the other PRs you're working on (web UI timeout and MOTD).